zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -369,33 +369,32 @@ <S extends ExporterEntity<S>, T extends RecordValue> void allHandlerTestsWithInv
         final ExportHandler<S, T> handler)
         throws IOException {
 
-      final var currentTime = OffsetDateTime.now().truncatedTo(ChronoUnit.MILLIS);
-      final var mocked = mockStatic(OffsetDateTime.class);
-      mocked.when(OffsetDateTime::now).thenReturn(currentTime);
-
-      final var record =
-          (Record<T>)
-              customRecordGenerators
-                  .getOrDefault(handler.getClass(), this::defaultRecordGenerator)
-                  .apply(handler);
-      customRecordGenerators.put(handler.getClass(), (ignored) -> record);
-
-      final var entityId = handler.generateIds(record).getFirst();
-      final var expectedEntity = handler.createNewEntity(entityId);
-      handler.updateEntity(record, expectedEntity);
-      exporter.export(record);
-
-      mocked.close();
-
-      final var responseEntity =
-          clientAdapter.get(
-              expectedEntity.getId(), handler.getIndexName(), handler.getEntityType());
-
-      assertThat(responseEntity)
-          .describedAs(
-              "Handler [%s] correctly handles a [%s] record",
-              handler.getClass().getSimpleName(), handler.getHandledValueType())
-          .isEqualTo(expectedEntity);
+      try (final var mocked = mockStatic(OffsetDateTime.class)) {
+        final var currentTime = OffsetDateTime.now().truncatedTo(ChronoUnit.MILLIS);
```

Comment:
:wrench: It's probably better to mock static in a `@BeforeAll` and close it in an `@AfterAll`,

Reply:
the test class currently doesn't have `BeforeAll` or `AfterAll`, and I don't want to introduce them in this PR just for that. But it's a plausible thought.

Reply:
sure